src/components/modal/Modal.vue

```diff
@@ -305,6 +306,7 @@ export default {
             document.documentElement.scrollTop = savedScrollTop
             document.body.style.top = null
         }
+        clearTimeout(this.timeOutID)
```

Comment:
@Arooba-git This should be unnecessary, because `timeOutID` is set only if `Modal` has programmatically been opened (`this.programmatic` is `true`) and programmatically opened `Modal` won't be destroyed until the timeout expires. Please see the line 245.

Reply:
@Arooba-git I have reverted the changes in your fork.